modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/dsl/Transaction.scala

```diff
@@ -132,8 +137,16 @@ object Transaction {
       def transact[A](io: ConnectionIO[A]): F[A] =
         DefaultTransactor.trans.apply(io)
```

Comment:
Should we use same bracket join pattern in here too ?

Reply:
Oh, definitely we need it yes! Thanks for spotting this, are you OK to do the amendment?

Reply:
Sure 👍